client/jest.config.js

```diff
@@ -0,0 +1,52 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: [
+    '**/src/components/**/*.{js}',
+    '!**/src/components/**/*{mock,stories,style,index}.{js}',
+    '!**/node_modules/**',
+    '!**/public/**',
+  ],
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: '<rootDir>/reports/coverage',
+
+  // Coverage Threshold to pass the test
+  coverageThreshold: {
+    global: {
+      branches: 80,
+      functions: 80,
+      lines: 80,
+      statements: 80,
+    },
+  },
+
+  // An array of file extensions your modules use
+  moduleFileExtensions: ['js', 'json', 'jsx'],
+
+  // The paths to modules that run some code to configure or set up the testing environment before each test
+  setupFiles: ['<rootDir>/enzyme.config.js'],
+
+  // The test environment that will be used for testing
+  testEnvironment: 'jsdom',
+
+  // The glob patterns Jest uses to detect test files
+  testMatch: ['**/__tests__/**/*.js?(x)', '**/?(*.)+(spec|test).js?(x)'],
+
+  // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
+  testPathIgnorePatterns: ['\\\\node_modules\\\\'],
```

Comment:
Should we need to add "build" folder here as well or not required?

Reply:
not required as it will search for the __tests__ or *.test pattern.